etc/adminhtml/system.xml

```diff
@@ -338,6 +340,30 @@
                     <comment><![CDATA[Customs Duty & Import Tax include duties, fees and taxes relevant to cross border shipments. This feature must be enabled in your Avalara account. Note that the proper use of this feature requires appropriate configuration of <strong>Cross Border Classes</strong> and assignment of Cross Border Types to products.]]></comment>
                 </field>
             </group>
+            <group id="avatax_document_management" translate="label" sortOrder="7" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
@LordZardeck - This stuff specific to Document Management shouldn't be in this PR.  (And, in fact, it's critical that it not be, because the other changes in this PR will get merged directly into release/2.0.0 for the next release, while Document Management as a whole may not be included in the next release.)